### PR TITLE
build more plugin related packages on el9

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: package_manifest.yaml [hunk 1]
```diff
@@ -721,18 +721,10 @@ foreman_plugin_packages:
   children:
     ruby_plugin_packages: {}
   hosts:
-    ansible-collection-theforeman-operations: {}
-    ansiblerole-foreman_scap_client: {}
-    ansiblerole-insights-client: {}
     ansible-runner: {}
-    ansible-collection-theforeman-foreman: {}
-    cjson: {}
     foreman-discovery-image-service: {}
```
Comment (ekohl) [inline package_manifest.yaml:725]: This is a weird one. I wonder if the discovery image should be its own category.

File: package_manifest.yaml [hunk 2]
```diff
@@ -721,18 +721,10 @@ foreman_plugin_packages:
   children:
     ruby_plugin_packages: {}
   hosts:
-    ansible-collection-theforeman-operations: {}
-    ansiblerole-foreman_scap_client: {}
-    ansiblerole-insights-client: {}
     ansible-runner: {}
```
Comment (ekohl) [inline package_manifest.yaml:724]: Isn't runner also only for the plugin?

File: packages/plugins/cjson/cjson.spec [hunk 3]
```diff
@@ -28,17 +28,15 @@ developing applications that use cJSON.
 
 %build
 %cmake
-make %{?_smp_mflags}
```
Comment (ekohl) [inline packages/plugins/cjson/cjson.spec:31]: Note to self: this reverts 5a4edfdf077e67e68f3995c2d8169011ed064112

File: packages/plugins/cjson/cjson.spec [hunk 4]
```diff
@@ -28,17 +28,15 @@ developing applications that use cJSON.
...
+%cmake_build
 
 %install
-%make_install
+%cmake_install
 rm -f %{buildroot}%{_libdir}/*.{la,a}
 rm -f %{buildroot}%{_libdir}/cmake/cJSON/*.cmake
 
 %check
-%if 0%{?rhel} > 7
```
Comment (ekohl) [inline packages/plugins/cjson/cjson.spec:39]: Note to self: this reverts d67803aaec2ab881c492abb52468c59c06d71510

File: packages/plugins/cjson/cjson.spec
Comment (ekohl) [inline packages/plugins/cjson/cjson.spec:1]: Any thoughts on syncing in the latest package from Fedora? Right now there's already 1.7.14-5 in EPEL9 with a certain history. While technically this change brings them back in line, I'd pull the latest spec file in, including the whole history.

File: package_manifest.yaml (hunk 2 above)
Comment (evgeni) [reply package_manifest.yaml:724]: yes, but I didn't want to build it just yet, so left it out for now

File: packages/plugins/cjson/cjson.spec (hunk 3 above)
Comment (evgeni) [reply packages/plugins/cjson/cjson.spec:31]: as noted in the commit message, yes :)

File: packages/plugins/cjson/cjson.spec
Comment (evgeni) [reply packages/plugins/cjson/cjson.spec:1]: I can do that. Sure.

Comment (evgeni) [reply packages/plugins/cjson/cjson.spec:1]: err. we already had 1.7.14-5 from epel9, but with our patches, so pulling that spec in *would* revert the changes, but the version would not increase :(

Comment (evgeni) [reply packages/plugins/cjson/cjson.spec:1]: (we could pull more changelog from fedora, but what's the point?)

File: package_manifest.yaml (hunk 1 above)
Comment (evgeni) [reply package_manifest.yaml:725]: a clear maybe (I first moved it to the "other" group too, and then was like "nah, this is a special one")

File: packages/plugins/cjson/cjson.spec
Comment (ekohl) [reply packages/plugins/cjson/cjson.spec:1]: I meant to pull in 1.7.15-2 from Fedora. That way, if they update EPEL9 to exactly the version from Fedora we will be in sync.

Comment (evgeni) [reply packages/plugins/cjson/cjson.spec:1]: Ah! Yeah, I can do that.

**Comment by evgeni** [comment]
welp, sometimes our linter is too smart.

**Comment by evgeni** [comment]
finally, :green_apple: again

File: packages/plugins/cjson/cjson.spec (hunk 3 above)
Comment (ekohl) [reply packages/plugins/cjson/cjson.spec:31]: I wish GitHub made those more obvious in but GH and multi commits in PRs remains sub-optimal.

**Review by ekohl — APPROVED** [review]

**Pull request merged**